community-tutorials/run-24-7-start-on-boot/01-en.md

```diff
@@ -0,0 +1,88 @@
+---
+title: run something 24/7 and start it on boot
+description: How to run something 24/7 and start it on boot Debian and Ubuntu servers
+updated_at: 2021-11-11
+slug: run-24-7-start-on-boot
+author_name: Jonas Löchner
+author_url: https://sancraft.dev/
+author_image: https://dl.sancraft.dev/images/Sanoj/1-B.png
+author_bio: -
+tags: [linux, debian, cron, boot, 24/7, python]
+netcup_product_url: https://netcup.de/bestellen/produkt.php?produkt=2623
+language: en
+available_languages: en
+---
+
+# Introduction
+This tutorial will show you, how to run something 24/7 and start it on boot Debian and Ubuntu servers <br>
+For this, i will use as example a python script <br>
+I am doing this on a debian 11 server (Netcup RS 2000), but it also should work on other debian versions and ubunt 
+
+# Requirements
+You need a Debian or Ubuntu Server with an Internet Connection <br>
+You also need root permissions on your server.
+
+# Step 1 - Installing Cron
+First you need to connect to your server over ssh. <br>
+Then we are going to switch to the root user and are going to update the system, and then we are going to install Cron (and python)
+```sh
+su
+apt update && apt upgrade -y && apt autoremove -y
+apt install wget curl sudo nano python-pip-whl cron python3 python3-pip screen -y
+```
+
+## Step 1.1 - Starting
+In may example i have a python script in /home/python, which is named run.py and i want to start it on boot and it should run 24/7 <br>
+First I go in the folder and install my python requirements that i need for the script
+```sh
+cd /home/python
+pip3 install py-cord py-cord[voice]
+```
+Then I am going to start it (Replace `discord-py-bot` with a name and `python3 run.py` with the command to start)
+```sh
+screen -AmdS discord-py-bot python3 run.py
```

Comment:
Could you please amend why we are using screen?

Reply:
done